js/aax.js

```diff
@@ -458,7 +457,7 @@ module.exports = class aax extends Exchange {
             const quote = this.safeCurrencyCode (quoteId);
             const settle = this.safeCurrencyCode (settleId);
             const status = this.safeString (market, 'status');
-            const marketType = this.safeString (market, 'type');
+            const marketType = this.safeString (market, 'accountsByType');
```

Comment:
I would suggest to keep `types` here, cause this is more unified than `accountsByType`. If possible, please, revert this bit. The same is true for `accounts`. Let's keep the options-names as is, but still add the code to support both the unified account names and the exchange-specific names + the uniform exception messages.

Reply:
Although, on a second thought, I'm not entirely sure if we should revert yet. Please, hold on with that, I will review the other PRs and will add more comments here.

Reply:
Sounds good, I only opened the refactor in 2 or 3 exchanges, assuming there may be some comments.